### PR TITLE
Add a link to solanahcl

### PR DESCRIPTION
**Pull request by ferric-sol**
#### Problem
Added a link to the solana hardware compatibility list https://github.com/solanahcl/solanahcl as suggested by @tigarcia @willhickey - I've also added both Tim and Will as collaborator/admins on the github repo, along with https://github.com/bartenbach and https://github.com/schmiatz and https://github.com/1000xsh from the validator community in case I get hit by a bus :)

#### Summary of Changes
Link to solanahcl.org's github repo

### CODE REVIEW

File: docs/src/operations/requirements.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The hardware recommendations below are provided as a guide.  Operators are encou
 | **Disk**  | PCIe Gen3 x4 NVME SSD, or better, on each of: <br />- **Accounts**: 500GB, or larger. High TBW (Total Bytes Written)<br />- **Ledger**: 1TB or larger. High TBW suggested<br />- **Snapshots**: 250GB or larger. High TBW suggested<br />- **OS**: (Optional) 500GB, or larger. SATA OK<br /><br />The OS may be installed on the ledger disk, though testing has shown better performance with the ledger on its own disk<br /><br />Accounts and ledger *can* be stored on the same disk, however due to high IOPS, this is not recommended<br /><br />The Samsung 970 and 980 Pro series SSDs are popular with the validator community | Consider a larger ledger disk if longer transaction history is required<br /><br />Accounts and ledger **should not** be stored on the same disk |
 | **GPUs**  | Not necessary at this time<br />Operators in the validator community do not use GPUs currently | |
 
+A community maintained list of currently optimal hardware can be found here: [solanahcl.org](https://github.com/solanahcl/solanahcl)
```
Comment (willhickey) [inline docs/src/operations/requirements.md]: I'd rather avoid the link text being a URL that isn't where the link points. What about something like this:

```suggestion
A community maintained list of currently optimal hardware can be found here: [solanahcl.org)[https://solanahcl.org/). It is updated automatically from the [solanahcl/solanahcl Github repo][https://github.com/solanahcl/solanahcl].
```

Comment (ferric-sol) [reply docs/src/operations/requirements.md]: Yep, that reads a lot better. Committed it, thank you!

File: docs/src/operations/requirements.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ The hardware recommendations below are provided as a guide.  Operators are encou
 | **Disk**  | PCIe Gen3 x4 NVME SSD, or better, on each of: <br />- **Accounts**: 500GB, or larger. High TBW (Total Bytes Written)<br />- **Ledger**: 1TB or larger. High TBW suggested<br />- **Snapshots**: 250GB or larger. High TBW suggested<br />- **OS**: (Optional) 500GB, or larger. SATA OK<br /><br />The OS may be installed on the ledger disk, though testing has shown better performance with the ledger on its own disk<br /><br />Accounts and ledger *can* be stored on the same disk, however due to high IOPS, this is not recommended<br /><br />The Samsung 970 and 980 Pro series SSDs are popular with the validator community | Consider a larger ledger disk if longer transaction history is required<br /><br />Accounts and ledger **should not** be stored on the same disk |
 | **GPUs**  | Not necessary at this time<br />Operators in the validator community do not use GPUs currently | |
 
+A community maintained list of currently optimal hardware can be found here: [solanahcl.org)[https://solanahcl.org/). It is updated automatically from the [solanahcl/solanahcl Github repo][https://github.com/solanahcl/solanahcl].
```
Comment (willhickey) [inline docs/src/operations/requirements.md]: Oh bother, I messed up the markdown. This should fix it:

```suggestion
A community maintained list of currently optimal hardware can be found here: [solanahcl.org](https://solanahcl.org/). It is updated automatically from the [solanahcl/solanahcl Github repo](https://github.com/solanahcl/solanahcl).
```

**Review by willhickey — APPROVED** [review]
Looks good. Thanks for the PR and for maintaining the hardware repo!

**Review by tigarcia — APPROVED** [review]
Looks good. Thanks for maintaining this!

**Pull request merged**